cmds/internet/main.go

```diff
@@ -215,6 +219,19 @@ func configureZOS() error {
 			}
 		}
 
+		if env.NTPServer != nil && *env.NTPServer != "" {
+			// override /etc/ntp.conf
+			servers := strings.Split(*env.NTPServer, ",")
+			var buffer strings.Builder
+			for _, server := range servers {
+				buffer.WriteString(fmt.Sprintf("server %s\n", server))
```

Comment:
The server should have space trimmed, but it won't matter given the whitespace tolerance in the ntp file